packages/frontend-2/lib/viewer/composables/serialization.ts

```diff
@@ -301,49 +313,11 @@ export function useApplySerializedState() {
     if (filters.hiddenObjectIds?.length) {
       resetFilters()
       hideObjects(filters.hiddenObjectIds, { replace: true })
-    } else if (filters.isolatedObjectIds?.length) {
-      resetFilters()
-      isolateObjects(filters.isolatedObjectIds, { replace: true })
-    } else {
-      resetFilters()
     }
 
-    const propertyFilterApplied = filters.propertyFilter?.isApplied
-    if (propertyFilterApplied) {
-      applyPropertyFilter()
-    } else {
-      unApplyPropertyFilter()
-    }
-
-    const propertyInfoKey = filters.propertyFilter?.key
-    const passMin = state.viewer?.metadata?.filteringState?.passMin
-    const passMax = state.viewer?.metadata?.filteringState?.passMax
-    if (propertyInfoKey) {
-      removePropertyFilter()
-
-      // Setting property filter asynchronously, when it's possible to do so
-      waitForAvailableFilter(propertyInfoKey)
-        .then((filter) => {
-          if (passMin || passMax) {
-            const numericFilter = { ...filter } as NumericPropertyInfo
-            numericFilter.passMin = passMin || numericFilter.min
-            numericFilter.passMax = passMax || numericFilter.max
-            setPropertyFilter(numericFilter)
-            applyPropertyFilter()
-          } else {
-            setPropertyFilter(filter)
-            applyPropertyFilter()
-          }
-        })
-        .catch((e) => {
-          if (e instanceof TimeoutError) {
-            logger.warn(
-              `${e.message} - filter probably comes from a thread context that isn't currently loaded`
-            )
-          } else {
-            logger.error(e)
-          }
-        })
+    // Restore propertyFilters
+    if (filters.propertyFilters?.length) {
+      restoreFilters(filters.propertyFilters)
```

Comment:
what about the `else` branch? if you currently have filters and the incoming state doesnt have any, we still want to set filters to `[]`